cypress/plugins/tasks.js

```diff
@@ -13,10 +13,20 @@
 // limitations under the License.
 
 const { execSync } = require('child_process')
+const fs = require('fs')
 
+const { Client } = require('pg')
 const yaml = require('js-yaml')
 const codeCoverageTask = require('@cypress/code-coverage/task')
 
+const client = new Client({
+  user: 'root',
+  host: 'localhost',
+  database: 'ttn_lorawan_dev',
```

Comment:
not for this PR, but this should be passed from `mage`. otherwise its hardcoded twice in two different places